extension/src/pipeline/quickPick.test.ts

```diff
@@ -580,6 +580,35 @@ describe('pickPlotConfiguration', () => {
     expect(mockedQuickPickUserOrderedValues).toHaveBeenCalledTimes(2)
     expect(noFieldsResult).toStrictEqual(undefined)
   })
+
+  it('should show a toast if user selects more than one key in a file for an x field', async () => {
```

Comment:
At this point, we've got around 20 different tests in this `describe` block. I'd like to break group these tests to make things more readable but I'll do it in a separate pr since that would involve touching the whole file. Updated #4654.

Reply:
[Q] What is the problem with having 20 tests in a block?

Reply:
Not necessarily a problem but I find it easier to navigate/update a test file when tests are grouped. Looking at this file, we could group file picking and field picking into its own blocks. Though, of course, no strong preference, I'm happy to leave it as is if preferred.